docs/UserGuide.md

```diff
@@ -659,6 +656,8 @@ New session added: Tutorial 4 @ 12/07/2020
 
 **:information_source: Notes about the usage of this command:**<br>
 * Adds a session with the specified name and date to the displayed session list.
+* `SESSION_DATE` should be given in the format of d/M/yyyy. This means if your day or month is single digit, you do not
```

Comment:
Perhaps can specify `SESSION_DATE` can be given in the format of dd/MM/yyyy or d/M/yyyy ?